products/compute/terraform.yaml

```diff
@@ -1385,7 +1385,6 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "region_health_check_http_logs"
         primary_resource_id: "http-region-health-check"
-        min_version: beta
```

Comment:
This example relies on the beta-only field `log_config`, which causes the example to fail

Reply:
Thanks for catching this and apologies for blindly removing the min_version flag here.